web/packages/dmt-core/src/components/UiPluginSelector.tsx

```diff
@@ -172,6 +174,7 @@ export function UIPluginSelector(props: {
   const [blueprint, loadingBlueprint, error] = useBlueprint(type)
   // @ts-ignore
   const { loading, getUiPlugin } = useContext(UiPluginContext)
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore